xivo_bus/tests/test_marshaler.py

```diff
@@ -82,6 +81,14 @@ def test_given_valid_message_when_unmarshal_message_then_return_object(self):
         assert_that(result.value, equal_to(1234))
         assert_that(result.metadata, equal_to({'origin_uuid': 'some-origin'}))
 
+    @patch("xivo_bus.marshaler.datetime")
```

Comment:
Consistency: we try to use the same string quote style across the code. In this case, it should be single quotes (`'str'`)